docker/pytorch/inference/1.13.1/Dockerfile.neuronx

```diff
@@ -51,13 +51,14 @@ RUN apt-get update \
     unzip \
     zlib1g-dev \
     libcap-dev \
+    gnupg2 \
     gpg-agent \
  && rm -rf /var/lib/apt/lists/* \
  && rm -rf /tmp/tmp* \
  && apt-get clean
 
 RUN echo "deb https://apt.repos.neuron.amazonaws.com focal main" > /etc/apt/sources.list.d/neuron.list
-RUN wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB | apt-key add -
+RUN wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB | gpg --dearmor -o /etc/apt/trusted.gpg.d/aws-neuron-repository.gpg
```

Comment:
i dont remember these changes. can you verify where are these coming from?

Reply:
Removed